### ISSUE TITLE
[FEATURE] [MODDED] Clothing items inherit some aspects from CDDA, such as sizing and "poor fit" statuses

### ISSUE THREAD

**Issue by VoyagerSoa** [VoyagerSoa/jad_candlelightMoonfall#3]
This will be part of a greater fork of the existing mod on the Workshop. Rarifies the most common loot table (clothing from zombies) to be more dynamic as well as introducing new gameplay for tailoring.

**Comment by VoyagerSoa** [comment]
Here are my requirements:
- Same functionality as the existing mod, but introduced gameplay for Tailoring that will let them dynamically resize clothing with needle, thread, and applicable textiles (either cloth, denim, or leather strips).
- At a high enough Tailoring level, tailors may pad clothing without affecting the clothing's texture, so they can stop looking like vagrants for the sake of more protection.

**Comment by VoyagerSoa** [comment]
Sounds good. I know the mod introduces "gameplay" in the sense that you have to check tags to know what size an article of clothing is (until a certain Tailoring level). I do not think this is particularly realistic (most people can tell if a shirt or pant will most probably fit them on immediate visual inspection) so if you could remove that functionality to make sizes transparent regardless of skill level (perhaps generalizing it at low Tailoring levels? "This article of clothing will probably/probably not fit you" versus a totally accurate size readout)

**Comment by VanorFeadiel** [comment]
Yep the above has been remove in my version of the mod, the size can be looked at by anyone, is much quicker than the base mod and you can't cut the label.

**Comment by VanorFeadiel** [comment]
I mean they still have to look at it, I could change it to assess size, instead of check label in the menu, but it's much quicker.  I will upload what I have on this one and Realistic Inventory so far later today.
